src/widgets/ASlider.cpp

```diff
@@ -480,12 +480,17 @@ LWSlider::LWSlider(wxWindow * parent,
                      float stepValue,
                      bool canUseShift,
                      int style,
+                     bool showlabels /* = true */,
```

Comment:
Misspelled “without” in the commit comment

Reply:
Done.  Corrected spelling mistake.